tests/lib/ocpp/common/test_message_queue.cpp

```diff
@@ -0,0 +1,362 @@
+#include <gmock/gmock.h>
```

Comment:
This is missing the license header
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
#include <gmock/gmock.h>
```